awx/ui_next/src/screens/Schedule/ScheduleList/ScheduleList.jsx

```diff
@@ -0,0 +1,205 @@
+import React, { useState, useEffect } from 'react';
+import { useLocation } from 'react-router-dom';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import { SchedulesAPI } from '@api';
+import { Card, PageSection } from '@patternfly/react-core';
+import AlertModal from '@components/AlertModal';
+import ErrorDetail from '@components/ErrorDetail';
+import DataListToolbar from '@components/DataListToolbar';
+import PaginatedDataList, {
+  ToolbarDeleteButton,
+} from '@components/PaginatedDataList';
+import { getQSConfig, parseQueryString } from '@util/qs';
+import { ScheduleListItem } from '.';
+
+const QS_CONFIG = getQSConfig('schedule', {
+  page: 1,
+  page_size: 20,
+  order_by: 'unified_job_template__polymorphic_ctype__model',
+});
+
+function ScheduleList({ i18n }) {
+  const [contentError, setContentError] = useState(null);
+  const [scheduleCount, setScheduleCount] = useState(0);
+  const [schedules, setSchedules] = useState([]);
+  const [deletionError, setDeletionError] = useState(null);
+  const [hasContentLoading, setHasContentLoading] = useState(true);
+  const [selected, setSelected] = useState([]);
+  const [toggleError, setToggleError] = useState(null);
+  const [toggleLoading, setToggleLoading] = useState(null);
+
+  const location = useLocation();
+
+  const loadSchedules = async ({ search }) => {
```

Comment:
if you want to convert this to the new `useRequest` hook you can -- I think that landed just after this PR went up. Otherwise I'm flushing through all lists currently and I can update it post merge, so no big deal either way

Reply:
Ah ok - yep I'll do that